### PR TITLE
Fix admin midis not being able to be queued

### PR DESCRIPTION
**Pull request by lzimann**
Same as #16540. But this time merging into master instead of aprils fool. Hopefully it works.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
:-1: queuing midis is in really bad taste anyways, leave it not working

File: code/modules/admin/verbs/playsound.dm [hunk 1]
```diff
@@ -7,29 +7,27 @@ var/sound/admin_sound
...
-
+		
 	for(var/mob/M in player_list)
 		if(M.client.prefs.toggles & SOUND_MIDI)
 			var/vol = M.client.prefs.adminmusicvolume
-			M << sound(admin_sound,channel = SOUND_CHANNEL_ADMIN,volume=vol)
-
-	admin_sound.frequency = 1 //Remove this line when the AFD stuff above is gone
-	admin_sound.wait = 0
+			var/sound/admin_sound = new()
```
Comment (RemieRichards) [inline code/modules/admin/verbs/playsound.dm]: this is creating the sound for every player.
so for 50 players, 50 sounds.
I do not like this.

Comment (MrStonedOne) [inline code/modules/admin/verbs/playsound.dm]: They already do that, as the old M << sound(admin_sound...) creates a new sound datum.

=P

Comment (RemieRichards) [inline code/modules/admin/verbs/playsound.dm]: reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

**Comment by RemieRichards** [comment]
This is good now.

**Pull request merged**